apps/cli/src/commands/build.ts

```diff
@@ -109,9 +109,9 @@ export default class BuildApplication extends Command {
             dataSize: labels[CARTESI_LABEL_DATA_SIZE] ?? "10Mb",
             entrypoint: imageInfo["Config"]["Entrypoint"] ?? [],
             env: imageInfo["Config"]["Env"] || [],
-            ramSize: labels[CARTESI_LABEL_RAM_SIZE] ?? DEFAULT_RAM_SIZE,
+            ramSize: labels[CARTESI_LABEL_RAM_SIZE] ?? CARTESI_DEFAULT_RAM_SIZE,
             sdkVersion:
-                labels[CARTESI_LABEL_SDK_VERSION] ?? DEFAULT_SDK_VERSION,
+                labels[CARTESI_LABEL_SDK_VERSION] ?? SUNODO_DEFAULT_SDK_VERSION,
```

Comment:
This is not gonna work great. `CARTESI_SDK_VERSION` and `SUNODO_SDK_VERSION` have different values.

If we wan't to label the image to have the information about which version of the CARTESI_SDK it's using, we should keep a it locked into `0.15.0`, and use the `SUNODO_SDK_VERSION`  to track which `sunodo/sdk` will be used.

Reply:
I think the label should be a sunodo label, not a cartesi label, because it tracks the sunodo sdk version.